renderer/components/DateSelector.tsx

```diff
@@ -31,7 +32,9 @@ export default function DateSelector({
           </time>
         </h1>
         <p className="mt-1 text-sm text-gray-500">
-          {selectedDate?.toLocaleDateString("en-US", { weekday: "long" })}
+          {selectedDate.getDay() === today.getDay()
+            ? "Today"
+            : selectedDate?.toLocaleDateString("en-US", { weekday: "long" })}
```

Comment:
i think it would be more convenient to show weekday + note "today". I already asked this from the project lead